[ADD] web_widget_json_graph: Module to Draw json fields with graphs

luisg123v opened the pull request:
This module allows to load a line graph per ordered pair from an
One2many or
Many2many field.

![Widget in action](https://user-images.githubusercontent.com/19801181/99295574-35f60900-280b-11eb-9576-05466915cb12.png)

marcelsavegnago commented on web_widget_json_graph/__manifest__.py:
```diff
@@ -0,0 +1,24 @@
+    "author": "Vauxoo, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "category": "Hidden/Dependency",
+    "website": "https://github.com/OCA/web/tree/12.0/web_widget_json_graph",
```
```suggestion
    "website": "https://github.com/OCA/web",
```

luisg123v replied:
@marcelsavegnago done. However, AFAIK they're equivalent, at least that's [what the template says](https://github.com/OCA/maintainer-tools/blob/0290f3971ed057ff868731ddac36a28cf4b1bc97/template/module/__manifest__.py#L10)

luisg123v commented:
@moylop260 @hugho-ad @frahikLV,

Could you review, please?

moylop260 reviewed (approved):
👍

moylop260 commented:
@marcelsavegnago @hugho-ad

Could you check it, please?

marcelsavegnago reviewed (approved):
LGTM

frahikLV reviewed (approved):
+1 @luisg123v

hugho-ad commented on web_widget_json_graph/static/src/js/web_widget_json_graph.js:
```diff
@@ -0,0 +1,55 @@
+        ],
+        _render: function () {
+            var info = JSON.parse(this.value);
+            //this.$el.html(QWeb.render('JSONGraph', {}));
```
remove this commented code

moylop260 commented:
/ocabot merge nobump

hugho-ad commented on web_widget_json_graph/static/src/less/web_widget_json_graph.less:
```diff
@@ -0,0 +1,25 @@
+.oe_form_field_json_graph {
```
I think .less should be migrated to .sccs

hugho-ad replied:
in fact, this file is not used, remove it please

MeetKD commented on web_widget_json_graph/__manifest__.py:
```diff
@@ -0,0 +1,24 @@
+        "views/assets.xml",
+    ],
+    "demo": [
+    ],
```
@luisg123v Can you please remove empty keys ?

MeetKD commented on web_widget_json_graph/readme/CONTRIBUTORS.rst:
```diff
@@ -0,0 +1,5 @@
+* José Robles <josemanuel@vauxoo.com>
+* Luis González <lgonzalez@vauxoo.com>
+* Nhomar Hernández <nhomar@vauxoo.com>
+
```
@luisg123v You can also remove the empty lines

MeetKD commented on web_widget_json_graph/static/src/js/web_widget_json_graph.js:
```diff
@@ -0,0 +1,55 @@
+    var core = require('web.core');
+    var AbstractField = require('web.AbstractField');
+    var field_registry = require('web.field_registry');
+    var QWeb = core.qweb;
```
@luisg123v You can also remove unused Qweb variable.

luisg123v replied:
Done, also removed core as it's neither used.

MeetKD reviewed (changes requested):
Some non blocking comments.

luisg123v committed:
[ADD] web_widget_json_graph: Module to Draw json fields with graphs

This module allows to load a line graph per ordered pair from an
One2many or
Many2many field.

Co-Authored-By: Francisco Luna <fluna@vauxoo.com>
Co-Authored-By: José Robles <josemanuel@vauxoo.com>
Co-Authored-By: Luis González <lgonzalez@vauxoo.com>
Co-Authored-By: Nhomar Hernández <nhomar@vauxoo.com>

luisg123v commented:
Hi @hugho-ad @MeetKD,

Suggested changes have been applied, thanks! Could you check again, please?

MeetKD reviewed (approved):
Looks Good

hugho-ad commented:
LGMT :+1:

moylop260 commented:
/ocabot merge nobump

luisg123v commented:
@moylop260 it seems the bot got stuck during last command. Could you retry, please?

moylop260 commented:
/ocabot merge nobump

pedrobaeza commented:
The problem is that we have exhausted the free credits from Travis since the pricing change. There's an ongoing conversation in the contributors list about this.

moylop260 commented:
Thank you @pedrobaeza 
So, I just merge it since that it is a new module approved and the CI result are green
